packages/governance/README.md

```diff
@@ -70,32 +70,37 @@ voterHandle.
 
 ## ContractGovernor
 
-We want some contracts to be able to make it visible that their internal
-parameters are controlled by a public process, and allow observers to see who
-has control, and how those changes can happen. To do so, the contract will
-use a ParamManager to hold its mutable state. ParamManager has facets for
-accessing the param values and for setting them. The governed contract would use
-the access facet internally, and make that visible to anyone who should be able
-to see the values, while ensuring that the private facet, which can change the
-values, is only accessible to a visible ContractGovernor. The ContractGovernor
-makes the Electorate visible, while tightly controlling the process of
-creating new questions and presenting them to the electorate.
-
-The governor starts up the Contract and can see what parameters are subject to
-governance. It provides a private facet that carries the ability to request
-votes on changing particular parameters. Some day we may figure out how to make
-the process and schedule of selecting parameter changes to vote on also subject
-to governance, but that's too many meta-levels at this point.
+We want some contracts to be able to make the manner of their governance
+legible, that is that changes to their parameters, and the ability to invoke
+some management methods are controlled by a public process. Observers (clients
+and voters) must be able to see who has control, and how those changes can
```

Comment:
Glad to see this definition of this important jargon. Consider:
> Contracts can provide aspects that are _governed_ by a public process.
> Specifically,
> - changes to a defined subset of parameters
> - invocation of a defined subsets of methods

Should we name those two aspects of governance?

> The governance package enforces _legibility_, meaning that observers 
> (clients and voters) are able to see who has control and how those changes
> can happen. To make control of parameters legible…

Reply:
done